building/build-debs/homeworld-admin-tools/src/verify.py

```diff
@@ -80,17 +79,19 @@ def check_keygateway():
 
 
 def check_ssh_with_certs(hostname=None):
-    config = configuration.Config.load_from_project()
+    config = configuration.get_config()
     if hostname is None:
         if config.keyserver is None:
             command.fail("no keyserver found")
         hostname = config.keyserver.hostname
     env = dict(os.environ)
-    del env["SSH_AUTH_SOCK"]
-    del env["SSH_AGENT_PID"]
+    if "SSH_AUTH_SOCK" in env:
+        del env["SSH_AUTH_SOCK"]
+    if "SSH_AGENT_PID" in env:
+        del env["SSH_AGENT_PID"]
     keypath = access.renew_ssh_cert()
     try:
-        result = subprocess.check_output(["ssh", "-i", keypath, "root@%s.%s" % (hostname, config.external_domain), "echo confirmed"], env=env)
+        result = subprocess.check_output(ssh.SSH_BASE + ["-i", keypath, "root@%s.%s" % (hostname, config.external_domain), "echo confirmed"], env=env)
```

Comment:
Minor: can use ssh.ssh_get_login. Also "echo confirmed" should be split into "echo" and "confirmed".

Reply:
I can't use it directly, because I have a hostname, not a node -- but that just requires a bit of tweaking.